### ISSUE TITLE
Plugins install error

### ISSUE THREAD

**Issue by hiromipaw** [lektor/lektor#596]
Hi,
I have packaged Lektor for Debian [https://packages.debian.org/buster/lektor] and I have encountered an error when installing plugins.
The bug report is available at: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=905147

I suppose what is happening is that lektor tries to install a plugin with the --user flag while also the --target flag is present.
Is the --target flag necessary? The plugin is installed just fine with the --user flag in the .local folder of the user home directory. This should be consistent with people using lektor with the usual setup instead that with the packaged version.

I'd rather not patch the Debian package if you would consider solving this in lektor instead.

What do you think?

**Comment by nixjdm** [comment]
Plugins installations are supposed to be specific to Lektor projects, not just a Lektor installation. I could have one Lektor installation, and two Lektor projects that use two different versions of the same plugin.

Maybe there's something we can change on our end, but I don't know what all the .deb does. I haven't dug into that yet. How does it install? Where do I go to see what the .deb is doing? I haven't made a .deb before. A user scoped pip install of Lektor is normally fine, and the installer script works too.

**Comment by nixjdm** [comment]
Also, once this is working we can update https://www.getlektor.com/docs/installation/ to mention this as an alternate installation method.

**Comment by paternal** [comment]
I reported the [original bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=905147). I think this bug is fixed now and can be closed (both this issue and the debian bug): I reproduced the steps described in the original bug, and everything is fine.

Since lektor crashed when installing python package, this bug might have been fixed by the upgrade of the [python3-pip](https://packages.debian.org/buster/python3-pip) debian package (version 9 when I reported this bug; version 18 now).
